recipes-bsp/opensbi/opensbi_0.9.bbappend

```diff
@@ -1,5 +1,9 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
+# Bump to the latest version of OpenSBI, this doesn't appear to be
+# required, but does improve the output
+SRCREV_beaglev-starlight-jh7100 = "54d7def6c254058f9458a0e26205b3c93a48bb42"
```

Comment:
Lets bump this in oe-core, I have tested qemu works fine with this rev. It will also fix 

```
ERROR: opensbi-0.9-r0 do_patch: Fuzz detected:

Applying patch 0001-Makefile-Don-t-specify-mabi-or-march.patch
patching file Makefile
Hunk #1 succeeded at 206 with fuzz 1 (offset 37 lines).
Hunk #2 succeeded at 219 with fuzz 1 (offset 37 lines).


The context lines in the patches can be updated with devtool:

    devtool modify opensbi
    devtool finish --force-patch-refresh opensbi <layer_path>

Don't forget to review changes done by devtool!
```

which is coming from a patch in oe-core.

Reply:
No worries, I almost didn't include this as it boots without it, but I don't see all of the OpenSBI prints without this.